packages/diagrams/frontend/sirius-components-diagrams/src/renderer/panel/experimental-svg-export/useExperimentalSvgExport.tsx

```diff
@@ -50,7 +42,7 @@ export const useExperimentalSvgExport = (): UseExperimentalSvgExport => {
   const { data: elementSvgExportHandlerContributions } = useData(svgExportIElementSVGExportHandlerExtensionPoint);
 
   const exportSvg = useCallback((callback: (dataUrl: string) => void) => {
-    const nodesBounds: Rect = getNodesBounds(reactFlow.getNodes());
+    const nodesBounds: Rect = reactFlow.getNodesBounds(reactFlow.getNodes());
```

Comment:
I get an error when I try to build locally this file:
```
src/renderer/panel/experimental-svg-export/useExperimentalSvgExport.tsx:45:41 - error TS2339: Property 'getNodesBounds' does not exist on type 'ReactFlowInstance<Node<NodeData>, Edge<EdgeData>>'.
@eclipse-sirius/sirius-components-diagrams:build:
@eclipse-sirius/sirius-components-diagrams:build:
const nodesBounds: Rect = reactFlow.getNodesBounds(reactFlow.getNodes());
```

Reply:
That's odd I don't have the issue and I followed the instruction of a warning in the console asking me to use `reactFlow.getNodesBounds`.
Could you try building with `npm run build` first, and also, can you check the version of `xyflow` you have in your `node_modules`?